### PR TITLE
New Hindi Translation json file added

### PR DESCRIPTION
**Pull request by prashantSj789**

### CODE REVIEW

**Comment by prashantSj789** [comment]
Sorry sir had previously forgot to commit the changes which I made

File: lib/main.dart [hunk 1]
```diff
@@ -89,6 +88,7 @@ Future<Unit> main() async {
         Locale('te', 'IN'),
         Locale('th', 'TH'),
         Locale('zh', 'TW'),
+        Locale('hi','IN'),
```
Comment (guyluz11) [inline lib/main.dart:91]: It already exists, there is no reason to move it

File: hi-IN.json
Comment (guyluz11) [inline hi-IN.json:1]: The file is in the wrong folder, it should be under `assets/translations/`

As you can see we already support `hi-IN`
https://github.com/CyBear-Jinni/cbj_app/blob/master/assets/translations/hi-IN.json

You can add missing translations,

Comment (prashantSj789) [reply hi-IN.json:1]: okay really sorry

**Comment by guyluz11** [comment]
Closing the PR for the pull requests tab to stay organized.
Please re-open the pr after playing the needed changes that I have mentioned

**Pull request closed without merging**